chemicaltoolbox/autodock_vina/prepare_box/calc_vina_box_params.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import rdShapeHelpers
+import argparse
+
+
+def get_params(options):
+    # make sure we have a mol file by initiating rdkit mol object from input
+    mol = Chem.MolFromMolFile(options.ligand_path)
+    if not mol:
+        raise IOError
+
+    # get rdkit conformer and compute x,y,z of top and bottom corner of confounding cuboid
+    conf = mol.GetConformer()
+    params = rdShapeHelpers.ComputeConfBox(conf)
+
+    # change tuples to arrays
+    coords1 = np.array(params[0])
+    coords2 = np.array(params[1])
+
+    # get the centre of the box
+    center = np.mean((coords1, coords2), axis=0)
+
+    # calculate box dimensions
+    dims = np.abs(coords1 - coords2) + (options.tolerance,) * 3
```

Comment:
we can remove the `(options.tolerance,) * 3` thing because this is the same as adding the buffers

Reply:
Resolved by: b046d0c685b7638818660cadfb7d1c16e7d97545